code/modules/mob/language.dm

```diff
@@ -428,6 +428,19 @@
 	flags = RESTRICTED | HIVEMIND
 	follow = 1
 
+
+/datum/language/terrorspider
+	name = "TerrorSpider"
```

Comment:
This is inconsistent with any other language name- It should be `Terror Spider` or have an actual lore-friendly name.